docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md

```diff
@@ -1,6 +1,26 @@
 ---
 title: Rollbacks
 ---
+## Additional Steps for Rollbacks with Rancher v2.6.4+
+
+In Rancher v2.6.4, the cluster-api module has been upgraded from v0.4.4 to v1.0.2 in which the apiVersion of CAPI CRDs are upgraded from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. This has the effect of causing rollbacks from Rancher v2.6.4 to any previous version of Rancher v2.6.x to fail because the previous version the CRDs needed to roll back are no longer available in v1beta1.
```

Comment:
```suggestion
Rancher v2.6.4 upgrades the cluster-api module from v0.4.4 to v1.0.2. Version v1.0.2 of the cluster-api, in turn, upgrades the apiVersion of CAPI CRDs from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. The `cluster.x-k8s.io` upgrade to v1beta1 causes rollbacks to fail when you attempt to move from Rancher v2.6.4 to any previous version of Rancher v2.6.x. This is because v1beta1 doesn't contain the CAPI CRD versions that the rollback is attempting to restore.
```